roles/KubevirtTemplateValidator/tasks/main.yml

```diff
@@ -107,7 +107,7 @@
     namespace: "{{ cr_info.metadata.namespace }}"
     status:
       observedVersion: "{{ operator_version }}"
-  when: "{{ available==true }}"
+  when: "available==true"
```

Comment:
I think the quotation marks are here by accident? I see you have removed them from similar statements

Reply:
yes, you are right. Fixed